ecomm-proj/src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
```suggestion
import React, { PureComponent } from 'react';
```

Avoiding line 4 `React.`